Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -734,4 +721,14 @@ function generatePassword(): string {
 
 function getUserProvisioningProfilesPath(): string {
     return tl.resolve(tl.getVariable('HOME'), 'Library', 'MobileDevice', 'Provisioning Profiles');
+}
+
+function splitIntoKeyValue(line: string): {key: string, value: string} {
+    const index: number = line.indexOf('=');
+
+    if (index) {
+        return {key: line.substring(0, index), value: line.substring(index + 1)};
```

Comment:
can you use `line.split('=')`?

Reply:
The value can include `=`. `"subject= /UID=E848ASUQZY/CN=iPhone Developer: Chris Sidi (7RZ3N927YF)/OU=DJ8T2973U7/O=Chris Sidi/C=US".split("=", 2)` returns `[ 'subject', ' /UID' ]`

Reply:
Ok, thanks.  I think it would be good to have a comment on the expected inputs so someone doesn't "refactor" this.